module/tests/automated.js

```diff
@@ -2,12 +2,20 @@ module("actionsheet");
 
 // In this test we call the example showAlert API method with an empty string
 // In the example API method an empty string will immediately call the error callback
-asyncTest("Attempt to show an alert with no text", 1, function() {
-	forge.actionsheet.showAlert("", function () {
-		ok(false, "Expected failure, got success");
+asyncTest("Display a dialog", 1, function() {
+	var success = function(){
+		ok(true, "You clicked a button!");
 		start();
-	}, function () {
-		ok(true, "Expected failure");
+	};
+	forge.internal.call('actionsheet.show', {
+		text: "Press a button!",
+		destructiveTitle: "Test!",
+		cancelTitle: "Another test."
+	}, function() {
+		forge.internal.addEventListener('actionsheet.destructive', success);
```

Comment:
how does this automated test work, i.e how does it choose which button to press?